src/config/dynamic-theme-fixes.config

```diff
@@ -1539,6 +1539,11 @@ img.Ha
 .gb_df
 img[src$="google_gsuite"]
 
+CSS
+::-webkit-scrollbar-thumb {
+    background-color: #424242 !important;
```

Comment:
Please check how it looks in Light mode.

Reply:
![](https://i.imgur.com/kddtqEG.png) Very weird it doesn't show the black bar asides I think some CSS property is setted that cause this.